source/StigData/Processed/RHEL-7-3.13.xml

```diff
@@ -6263,9 +6273,9 @@ If "timestamp_timeout" is set to a negative number, is commented out, or no resu
     </Rule>
     <Rule id="V-250314" severity="medium" conversionstatus="pass" title="SRG-OS-000324-GPOS-00125" dscresource="nxFile">
       <Contents>%wheel ALL=(ALL) TYPE=sysadm_t ROLE=sysadm_r ALL</Contents>
```

Comment:
based on the contents, this appears to no longer be STIG guidance

Old: 
%wheel ALL=(ALL) TYPE=sysadm_t ROLE=sysadm_r ALL
New
%{designated_group_or_user_name} ALL=(ALL) TYPE=sysadm_t ROLE=sysadm_r ALL